CRM/Multicompanyaccounting/Page/Company.php

```diff
@@ -0,0 +1,51 @@
+<?php
+use CRM_Multicompanyaccounting_ExtensionUtil as E;
```

Comment:
unused import

Reply:
this is already removed in one of the commits